oktaawscli/aws_auth.py

```diff
@@ -146,9 +153,46 @@ def write_sts_token(self, profile, access_key_id, secret_access_key, session_tok
 
         with open(self.creds_file, 'w+') as configfile:
             config.write(configfile)
-        print("Temporary credentials written to profile: %s" % profile)
+        sys.stderr.write("Temporary credentials written to profile: %s\n" % profile)
         self.logger.info("Invoke using: aws --profile %s <service> <command>" % profile)
 
+    def copy_to_default(self, profile):
+        """ Reads STS auth information from credentials file """
+        if not os.path.exists(self.creds_dir):
+            os.makedirs(self.creds_dir)
+        config = RawConfigParser()
```

Comment:
We should be using [SafeConfigParser](https://docs.python.org/2/library/configparser.html#ConfigParser.SafeConfigParser).

Reply:
We can do that, there are other locations that will also need to be changed to use SafeConfigParser such as in okta_auth_config.py